ios/RNTeads.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  RNTeads.swift
+//  react-native-teads-sdk-module
+//
+//  Created by Numa Maurin on 21/12/2022.
+//
+
+import Foundation
+import TeadsSDK
+import UIKit
+
+@objc(RNTeads)
+final class RNTeads: NSObject {
+    
+    @objc
+    func createInReadPlacement(_ pid : Float, settingsMap : NSDictionary, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
+        let data = try! json(from: settingsMap)
+        let decoder = JSONDecoder()
+        do {
```

Comment:
avoid any force unwrap: make sure by searching in the whole project that there is no `!` in swift sources
```suggestion
        do {
            let data = try json(from: settingsMap)
            let decoder = JSONDecoder()
```